gitrevise/todo.py

```diff
@@ -141,7 +141,7 @@ def edit_todos_msgedit(repo: Repository, todos: List[Step]) -> List[Step]:
         todos_text += step.commit.message + b"\n"
 
     # Invoke the editors to parse commit messages.
-    response = run_editor(
+    response = run_sequence_editor(
```

Comment:
I'm not sure whether it makes sense to use `GIT_SEQUENCE_EDITOR` for the `git revise -ie` command, as the format doesn't look like `git rebase`'s format anymore, due to the leading `++` and inline commit messages. I don't know how people usually configure their `GIT_SEQUENCE_EDITOR`, but I worry commands expecting a `git-rebase` formatted file might handle this case poorly.

```suggestion
    response = run_editor(
```

Reply:
Ah, I didn't realize `-e` ran a different format than `-i`. I saw the `+++` formatting stuff but didn't ever see it appear, so assumed it was something internal. Dumb assumption. You're right, it wouldn't make sense to run a normal sequence editor with that format.